source/material_model/interface.cc

```diff
@@ -281,8 +281,24 @@ namespace aspect
 
 
     template <int dim>
+    MaterialModelDerivatives<dim>::MaterialModelDerivatives(const unsigned int n_points,
+            const unsigned int n_comp)
+    {
+       dviscosities_du.resize(n_points, aspect::Utilities::signaling_nan<double>());
```

Comment:
Please move this into a member initializer list (i.e., what comes after the `:` in a constructor call). I know this is the same style as is currently used below in `MaterialModelOutputs`, where we should probably change it as well.